source/WindowPeek.cpp

```diff
@@ -950,6 +958,10 @@ void WindowPeek::MessageReceived(BMessage* e) {
                                     setup->SetSlideshowMode( e->what ); 
                                     MenuTick( e, true , true);
                                     break;
+
+    case PEEK_IMAGE_ZOOM_IN: imagePane->SetZoom(imagePane->GetZoom()+0.1); break;
+    case PEEK_IMAGE_ZOOM_OUT: imagePane->SetZoom(imagePane->GetZoom()-0.1); break;
```

Comment:
Shouldn't these be changed to multiply/divide by 1.25?

Reply:
Oh shoot, I forgot about this one because I focused so much on the scroll wheel zooming.